app/models/campaign.rb

```diff
@@ -0,0 +1,14 @@
+class Campaign < ApplicationRecord
+  validates :title, presence: true,
+                    uniqueness: true,
+                    length: { in: 10..30 },
+                    format: { with: /\A[a-zA-Z0-9\s]+\z/ }
+  validates :description, presence: true,
+                          length: { in: 30..2000 }
```

Comment:
Why do we need to validate description length?

Reply:
not to allow the user enter too long description. 2000 symbols is pretty much long.
Should I remove this validation?